diff -urN slim-1.3.0/Makefile slim-1.3.0.new/Makefile
--- slim-1.3.0/Makefile	2007-08-26 11:34:21.095490552 +0200
+++ slim-1.3.0.new/Makefile	2007-08-26 11:41:40.028528565 +0200
@@ -5,9 +5,9 @@
 #######################################################
 CXX=/usr/bin/g++
 CC=/usr/bin/gcc
-CFLAGS=-Wall -I. -I/usr/include/freetype2 -I/usr/include/freetype2/config -I/usr/include/libpng12 -I/usr/include
+override CFLAGS += -I. -I/usr/include/freetype2 -I/usr/include/freetype2/config -I/usr/include/libpng14 -I/usr/include
 CXXFLAGS=$(CFLAGS)
-LDFLAGS=-lXft -lX11 -lfreetype -lXrender -lfontconfig -lpng12 -lz -lm -lcrypt -lXmu -lpng -ljpeg
+override LDFLAGS += -lXft -lX11 -lfreetype -lXrender -lfontconfig -lpng14 -lz -lm -lcrypt -lXmu -lpng -ljpeg
 CUSTOM=-DHAVE_SHADOW
 ifdef USE_PAM
 LDFLAGS+= -lpam
@@ -33,7 +33,7 @@
 all: slim
 
 slim: $(OBJECTS)
-	$(CXX) $(LDFLAGS) $(OBJECTS) -o $(NAME)
+	$(CXX) $(OBJECTS) -o $(NAME) $(LDFLAGS) 
 
 .cpp.o:
 	$(CXX) $(CXXFLAGS) $(DEFINES) $(CUSTOM) -c $< -o $@
